### PR TITLE
Add David Alvarado as a CAPI team member

### PR DESCRIPTION
**Pull request by tjvman**
Co-authored-by: David Alvarado <alvaradoda@vmware.com>

### CODE REVIEW

**Comment by dsboulder** [comment]
We don't need to approve the request, WG leads can decide who plays what role in their WGs. We'd also like you to add David to the other WG definition file (sorry for the duplication for now!).

**Comment by Gerg** [comment]
Here is the RFC I was mentioning: https://github.com/cloudfoundry/community/pull/248

**Comment by dsboulder** [comment]
@Gerg You have to edit the charter definition as well. Then I can help get it merged, since it doesn't need TOC approval

**Comment by Gerg** [comment]
See https://github.com/cloudfoundry/community/pull/262

File: org/cloudfoundry.yml [hunk 1]
```diff
@@ -2925,6 +2926,7 @@ orgs:
             - MarcPaquette
             - sweinstein22
             - MerricdeLauney
+            - dalvarado
```
Comment (Gerg) [inline org/cloudfoundry.yml:2929]: Does David meet the criteria described in https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0006-approver-requirements.md?

**Pull request merged**